kalliope/settings.yml

```diff
@@ -42,6 +42,7 @@ speech_to_text:
   - houndify: # In the online documentation but Not implemented yet in the speech recognition lib ..?
       key: "7zj90T7qAV74OYXk4X4vI2Xhk7wPsJu4aEZ0G5Ll-BMmV1JGtFpCxtSH9SmTY4G3bpEJ7a5y_GTQid-CAKI6vw=="
       client_id: "lN4JXeaSticbSo9-llczbA=="
+  - cmusphinx
```

Comment:
should be commented because it is not provided by default some package are required